ui/tx/Noves/components/NovesActionCard.tsx

```diff
@@ -0,0 +1,114 @@
+import { Box, Hide, Popover, PopoverArrow, PopoverContent, PopoverTrigger, Show, Text, useColorModeValue } from '@chakra-ui/react';
+import type { FC } from 'react';
+import React from 'react';
+
+import lightning from 'icons/lightning.svg';
+import Icon from 'ui/shared/chakra/Icon';
+import TokenEntity from 'ui/shared/entities/token/TokenEntity';
+
+import type { NovesAction, NovesFlowViewItem } from '../utils/NovesGenerateFlowViewData';
+import NovesTokensCard from './NovesTokensCard';
+
+interface Props {
+  item: NovesFlowViewItem;
+}
+
+const NovesActionCard: FC<Props> = ({ item }) => {
+  const popoverBg = useColorModeValue('gray.700', 'gray.300');
+
+  const getTokenData = (action: NovesAction) => {
+    const name = action.nft?.name || action.token?.name;
+    const symbol = action.nft?.symbol || action.token?.symbol;
+
+    const token = {
+      name: name,
+      symbol: symbol?.toLowerCase() === name?.toLowerCase() ? undefined : symbol,
+      address: action.nft?.address || action.token?.address,
+    };
+
+    return token;
+  };
+
+  return (
+    <>
+      <Hide above="md">
+        <Box display="flex" gap={ 2 } cursor="pointer" flexWrap="wrap">
+          <Text fontWeight="700" >
+            { item.action.label }
+          </Text>
+          <Text fontWeight="500">
+            { item.action.amount }
+          </Text>
+          <TokenEntity
+            token={ getTokenData(item.action) }
+            noCopy
+            noSymbol
+            noLink
+            fontWeight="500"
+            color="link"
```

Comment:
Why can it be a link to the token page? Consider using the `onlySymbol` prop instead of `noSymbol`

Reply:
It may be cases were we don't have the token address, so we check if it's available